src/plugins/crypto/README.md

```diff
@@ -0,0 +1,57 @@
+- infos = Information about crypto plugin is in keys below
+- infos/author = Peter Nirschl <peter.nirschl@gmail.com>
+- infos/licence = BSD
+- infos/needs = 
+- infos/provides = storage
```

Comment:
Well it's not exactly bound to any files, as you can pass configuration and content payload in-memory. Are you ok with "filter"?